### ISSUE TITLE
Designs should be stored per cut-site

### ISSUE THREAD

**Issue by t-vaudry** [t-vaudry/ribosoft#206]
With the new accessibility score calculations, each Design should be ranked per cut-site.

Calculate all other scores for the overall Candidate, and then loop over the cut-site indices to calculate accessibility and store them individually as a Design.

Add a column in the Design Details page, to indicate the cut-site index. Additionally, add a filter option for cut-site. Optionally, add sorting if you think it makes sense (I don't think it does).
